src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/DSASignatureDescription.cs

```diff
@@ -19,15 +19,15 @@ public DSASignatureDescription()
 
         public sealed override AsymmetricSignatureDeformatter CreateDeformatter(AsymmetricAlgorithm key)
         {
-            var item = (AsymmetricSignatureDeformatter)CryptoConfig.CreateFromName(DeformatterAlgorithm);
+            var item = (AsymmetricSignatureDeformatter)CryptoConfig.CreateFromName(DeformatterAlgorithm!)!;
```

Comment:
seems to me that wither `DeformatterAlgorithm` or `CreateFromName` is badly annotated or we're missing checks somewhere here

Reply:
`DeformatterAlgorithm` looks correct - it has a public setter that doesn't check for the value being assinged.

`CreateFromName` also **can** return null.